code/game/objects/structures/lavaland/necropolis_tendril.dm

```diff
@@ -0,0 +1,100 @@
+//Necropolis Tendrils, which spawn lavaland monsters and break into a chasm when killed
+/obj/structure/spawner/lavaland
+	name = "necropolis tendril"
+	desc = "A vile tendril of corruption, originating deep underground. Terrible monsters are pouring out of it."
+
+	icon = 'icons/mob/nest.dmi'
+	icon_state = "tendril"
+
+	faction = list("mining")
+	max_mobs = 3
+	max_integrity = 250
+	mob_types = list(/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/tendril)
+
+	move_resist = INFINITY // just killing it tears a massive hole in the ground, let's not move it
+	anchored = TRUE
+	burn_state = LAVA_PROOF
+
+	var/gps = null
+	var/obj/effect/light_emitter/tendril/emitted_light
+
+/obj/structure/spawner/lavaland/goliath
+	mob_types = list(/mob/living/simple_animal/hostile/asteroid/goliath/beast/tendril)
+
+/obj/structure/spawner/lavaland/legion
+	mob_types = list(/mob/living/simple_animal/hostile/asteroid/hivelord/legion/tendril)
+
+GLOBAL_LIST_INIT(tendrils, list())
+
+/obj/structure/spawner/lavaland/Initialize(mapload)
+	. = ..()
+	emitted_light = new(loc)
+	gps = new /obj/item/gps/internal(src)
+	GLOB.tendrils += src
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/spawner/lavaland/LateInitialize()
```

Comment:
@Markolie 

I noticed this was commented out; I wasn't sure why at first, until I noticed it causes runtimes; changing it to a `LateInitialize` resolved the issue and properly flattens the turfs around it.